### PR TITLE
[TM-1087] Polygon Version (PD View Site Profile)

### PR DESCRIPTION
**Pull request by LimberHope**
[Task Link](https://gfw.atlassian.net/browse/TM-1087)

### CODE REVIEW

File: src/components/elements/Map-mapbox/Map.tsx [hunk 1]
```diff
@@ -262,6 +263,29 @@ export const MapContainer = ({
...
   };
 
+  const addGeometryVersion = async () => {
+    const polygonGeojson = await fetchGetV2TerrafundPolygonGeojsonUuid({
+      pathParams: { uuid: selectedPolyVersion?.poly_id as string }
+    });
+    drawTemporaryPolygon(
+      polygonGeojson?.geojson,
+      () => {
+        console.log(`Polygon with UUID: ${selectedPolyVersion?.poly_id}.`);
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/Map.tsx:273]: Is this needed? Seems like 🛅 debug logging.

File: src/components/elements/Map-mapbox/MapControls/EditControl.tsx [hunk 2]
```diff
@@ -15,42 +19,44 @@ const EditControl = ({ onClick, onSave, onCancel }: { onClick?: any; onSave?: an
...
+          id="buttonEditPolygon"
+          variant="white"
+          className="w-full py-2"
+          onClick={() => {
+            setIsEditing(true);
+            onClick();
+          }}
+        >
+          <Text variant="text-12-bold" className="leading-[normal] text-black">
+            Edit Polygon
```
Comment (roguenet) [inline src/components/elements/Map-mapbox/MapControls/EditControl.tsx:34]: Text in this component needs translation with `useT`

File: src/components/elements/MapPolygonPanel/VersionInformation.tsx [hunk 3]
```diff
@@ -1,30 +1,106 @@
...
+      const response = (await fetchGetV2SitePolygonUuidVersions({
+        pathParams: { uuid: editPolygon?.primary_uuid as string }
+      })) as SitePolygonsDataResponse;
+      const polygonActive = response?.find(item => item.is_active);
+      setEditPolygon({
+        isOpen: true,
+        uuid: polygonActive?.poly_id as string,
+        primary_uuid: polygonActive?.primary_uuid
+      });
+      displayNotification("Polygon version deleted successfully", "success", "Success!");
```
Comment (roguenet) [inline src/components/elements/MapPolygonPanel/VersionInformation.tsx:57]: Text in this component needs translation.

File: src/components/elements/MapPolygonPanel/VersionInformation.tsx [hunk 4]
```diff
@@ -43,48 +126,85 @@ const VersionInformation = ({ setPreviewVersion }: { setPreviewVersion: Dispatch
...
+          </Text>
+          <div className="flex justify-between">
+            <button
+              className={classNames("text-10-bold h-min w-[64%] rounded-md border border-white", {
+                "bg-white text-[#797F62]": !!item.is_active,
+                "bg-transparent text-white": !item.is_active
+              })}
+              onClick={() => makeActivePolygon(item)}
+            >
+              {item.is_active ? "Yes" : "No"}
```
Comment (roguenet) [inline src/components/elements/MapPolygonPanel/VersionInformation.tsx:191]: `t()`

File: src/pages/site/[uuid]/components/SiteArea.tsx [hunk 5]
```diff
@@ -14,71 +18,147 @@ interface SiteAreaProps {
...
+      setPreviewVersion(false);
+      setOpenModalConfirmation(false);
+      setEditPolygon?.({
+        isOpen: true,
+        uuid: selectedPolyVersion?.poly_id as string,
+        primary_uuid: selectedPolyVersion?.primary_uuid
+      });
+    },
+    onError: () => {
+      displayNotification("Error making polygon version active", "error", "Error!");
```
Comment (roguenet) [inline src/pages/site/[uuid]/components/SiteArea.tsx:49]: `t()` on these notifications

**Review by pachonjcl — APPROVED** [review]

**Pull request merged**